manifests/overlays/dev/kustomization.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base
-  - ../../crd
-  - ./nvidia-dcgm-mock-exporter
+  - odh-dashboard-config.yaml
```

Comment:
Be careful with this, I think this is not needed for the PR right?

Reply:
Oops I think these were added due to rebasing, removing them.